.github/workflows/deploy-prod&demo.yaml

```diff
@@ -7,11 +7,29 @@ on:
   workflow_dispatch:
 
 jobs:
+  run-tests:
```

Comment:
Trenger vi ha testene i prod-filen?
Det at de er påkrevd i staging-deploy er vel tilstrekkelig, bedre å lage branch rule om at staging-deploy har kjørt grønt?

Reply:
For å opprettholde en coverage-fil for main-branch, ja.

Og koden i en branch vil ikke alltid være helt lik main. Det går jo som regel bra, men er alltids en sjangs for at noe plutselig ikke oppfører seg som forventet etter en merge.

Så føler ikke det er spesielt negativt å kjøre testene på main-branch før en prodsetting...